config/routes.rb

```diff
@@ -3,12 +3,18 @@
   root 'sessions#new'
 
   get 'login', to: 'sessions#new'
-  post 'login', to: 'sessions#create'
+  post 'login', to: 'sessions#create', as: 'post_login'
   get 'logout', to: 'sessions#destroy'
 
   get 'signup', to: 'users#new'
   post 'signup', to: 'users#create'
 
+  resources :users do
+    member do
+      get :confirm_email
+    end
+  end
+
```

Comment:
Sorry, little confusing... 

what's `member do`?
Why do we also have a `get :confirm_email` below?